src/components/Category/Category.jsx

```diff
@@ -1,7 +1,16 @@
 import "./Category.css";
+import { useCategory } from "../../context";
 
 export const Category = ({ category }) => {
-  const { categoryName } = category;
+  const { categoryName, videoCategory } = category;
+  const { categoryDispatch, selectedCategory } = useCategory();
 
-  return <button class="button chip">{categoryName}</button>;
+  return <button className={`${
+    videoCategory === selectedCategory ? "selected" : ""
+  } button chip`} onClick={() =>
+    categoryDispatch({
+      type: "CATEGORY",
+      payload: videoCategory
+    })
```

Comment:
suggestion: have another function say `handleClick` and inside that put all this
and give `handleClick` to onClick

Reply:
thanks, updated!!